### DISCUSSION TITLE
Tab UI placement

### DISCUSSION THREAD

**Discussion by Buzzardarg** [marimo-team/marimo#2860] [Ideas]
Hello, I was wondering if it would be possible to have an option for marimo.ui.tabs allowing to change the placement of the tabs. For example, I have three tabs, each one hasta a text area element below but in my case it would be tidier to have the tabs below the text areas, much like in Excel spreadsheets. Perhaps there could even be an option for left and right placement.

**Comment by mscolnick** [comment]
yes you can. the tabs are ui elements themselves, so you can use the selected tab to render the correct tab panel.

```python
tabs = mo.ui.tabs({"1": "", "2": "", "3": ""})
```

```python
def render_tab(tab: str):
   if tab == '1':
     ....
```

```python
mo.vstack([render_tab(tabs.value), tabs])
```
